web/src/containers/QuickExchange/index.tsx

```diff
@@ -80,6 +85,10 @@ export const QuickExchangeContainer = () => {
         const seconds = +msPricesUpdates() / 1000;
         setTime(ssToMMSS(seconds));
 
+        if (abilities.read && !CanCan.checkAbilityByAction('read', 'QuickExchange', abilities)) {
```

Comment:
better to check if abilities has been successfully fetched because [read] might be empty, while [update] might be fulfilled